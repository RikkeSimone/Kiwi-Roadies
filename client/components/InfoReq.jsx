import React from 'react'
import { connect } from 'react-redux'
import Greeting from './Greeting'
import RoadieForm from './RoadieForm'
import RoadieMap from './RoadieMap'

class InfoReq extends React.Component {
  state = { isLoading: true }

  componentDidMount () {
    this.setState({
      isLoading: false
    })
  }

  render () {
    return (
      <div>
        <Greeting />
        <RoadieForm />
        <RoadieMap />
      </div>
    )
  }
}

export default connect()(InfoReq)
