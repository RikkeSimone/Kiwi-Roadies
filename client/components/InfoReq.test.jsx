import React from 'react'
import { renderWithRedux } from '../testing/utils'
import '@testing-library/jest-dom'
import 'regenerator-runtime/runtime'

import InfoReq from './InfoReq'

test('InfoReq renders the expected items', async () => {
  const { asFragment } = renderWithRedux(<InfoReq />)
  expect(asFragment()).toMatchSnapshot()
})
