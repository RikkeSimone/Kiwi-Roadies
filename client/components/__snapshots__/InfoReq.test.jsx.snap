// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoReq renders the expected items 1`] = `<DocumentFragment />`;
